services.json

```diff
@@ -0,0 +1,44 @@
+{
+        "name": "contivModel",
+        "objects": [
+                {
+                 "name": "serviceLB",
+                 "type": "object",
+                 "key": ["serviceName","tenantName"],
+                 "properties": {
+                                "serviceName": {
+                                             "type": "string",
+                                             "title": "service name",
+                                             "length": 256
+                                },
+                                "tenantName": {
+                                        "type": "string",
+                                        "title": "Tenant Name",
+                                        "length": 64
+                                },
+
+                                "network": {
+                                             "type": "string",
+                                             "title": "Service subnet",
+                                             "length": 15
+                                },
+                                "ipAddress":{
+                                            "type":"string",
+                                            "title":"Service ip",
+                                            "length": 15
+                                },
+                                "labels":       {
+                                             "type": "array",
+                                             "title": "labels key value pair",
+                                             "length": 512,
+					     "items" :"string"
+                                },
+                                "ports":{
```

Comment:
@shaleman ports is of the format "svcport:provport:protocol" and this can be multiple entries.